index.html

```diff
@@ -52,6 +52,195 @@ <h4 class="social-title">LET'S CONNECT</h4>
           </li>
         </ul>
       </section>
+
+      <div class="works">
```

Comment:
- [ ] It is required to use `<section>` tag in this place instead of ordinary `<div>` for semantic reason.

Reply:
Yeah okay